src/hexen/p_map.c

```diff
@@ -1450,6 +1450,24 @@ void P_SlideMove(mobj_t * mo)
     }
 }
 
+//============================================================================
+//
+// P_InitSlideLine
+//
+//============================================================================
+
+void P_InitSlideLine(void)
+{
+    // use relevant parts from first bestslideline in demo1 (hexen.wad)
+    static vertex_t initvertex1 = {-77594624,37748736};
+    static line_t initslideline = {&initvertex1, NULL, 0, 6291456, 0, 0, 0, 0, 0, 0, 0, 
```

Comment:
I think most of our code has spaces between the braces and first/last value and space after commas.